www/src/views/shared/collapsible.js

```diff
@@ -1,83 +1,75 @@
 /** @jsx jsx */
 import { jsx } from "theme-ui"
-import { Component } from "react"
+import { useState } from "react"
 import { FaAngleDown, FaAngleUp } from "react-icons/fa"
 
-class Collapsible extends Component {
-  state = {
-    collapsed: false,
-  }
+import { Flex } from "theme-ui"
 
-  handleClick = () => {
-    this.setState({ collapsed: !this.state.collapsed })
-  }
+export default function Collapsible(props) {
+  const { heading, fixed, children } = props
```

Comment:
nit: prefer destructuring the props in the argument list.

```suggestion
export default function Collapsible({ heading, fixed, children }) {
```

Reply:
Working on it